ci/ray_ci/pipeline/gap_filling_scheduler.py

```diff
@@ -29,7 +30,7 @@ def __init__(
         self.repo_checkout = repo_checkout
         self.days_ago = days_ago
 
-    def run(self) -> List[str]:
+    def run(self) -> List[Optional[str]]:
```

Comment:
why is this returning a list of optional string?

can we just convert `None` to empty string and drop the `Optional`?